ext/mysqlnd/mysqlnd_structs.h

```diff
@@ -206,7 +206,7 @@ typedef struct st_mysqlnd_infile
 
 typedef struct st_mysqlnd_session_options
 {
-	ulong		flags;
+	zend_ulong		flags;
```

Comment:
Assuming flags fit, unsigned int would be better here. It doesn't really make sense to use a system-dependent integer type for flags.